tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -36,39 +36,17 @@ Add public prerelease access list
     user clicks button  Save access list
     user waits until element contains  css:[data-testid="publicPreReleaseAccessListPreview"]  Test public access list
 
-Go to "Sign off" page
+Go to "Sign off" page and approve release
     [Tags]  HappyPath
-    user clicks link   Sign off
```

Comment:
`user approves release for immediate publication` doesn't navigate to the Sign off page, so I think this is going to fail currently